app/models/manageiq/providers/ibm_cloud/container_manager/refresh_worker.rb

```diff
@@ -0,0 +1,8 @@
+class ManageIQ::Providers::IbmCloud::ContainerManager::RefreshWorker < ManageIQ::Providers::BaseManager::RefreshWorker
+  require_nested :Runner
+  require_nested :WatchThread
+
+  def self.settings_name
+    :event_catcher_ibm_cloud_iks
```

Comment:
Just noticed this should be `:ems_refresh_worker_ibm_cloud_iks` can you fix in a follow-up @nasark ?